src/modules/Dropdown/Dropdown.js

```diff
@@ -5,18 +5,103 @@ import React, {Component, PropTypes} from 'react';
 import META from '../../utils/Meta';
 import getUnhandledProps from '../../utils/getUnhandledProps';
 
+/* eslint react/jsx-sort-prop-types:0 */
 export default class Dropdown extends Component {
   static propTypes = {
+    // Settings
+    action: PropTypes.string,
+    allowAdditions: PropTypes.bool,
+    allowCategorySelection: PropTypes.bool,
+    apiSettings: PropTypes.object,
     className: PropTypes.string,
+    fields: PropTypes.object,
+    forceSelection: PropTypes.bool,
+    glyphWidth: PropTypes.number,
+    label: PropTypes.object,
+    match: PropTypes.string,
+    maxSelections: PropTypes.number,
+    on: PropTypes.string,
     options: PropTypes.arrayOf(PropTypes.shape({
       value: PropTypes.string,
       text: PropTypes.string,
     })),
+    placeholder: PropTypes.oneOf([
+      PropTypes.string,
+      PropTypes.bool,
+    ]),
+    saveRemoteData: PropTypes.bool,
+    useLabels: PropTypes.bool,
+
+    // Additional Settings
+    direction: PropTypes.string,
+    keepOnScreen: PropTypes.bool,
+    context: PropTypes.object,
+    fullTextSearch: PropTypes.bool,
+    preserveHTML: PropTypes.bool,
+    sortSelect: PropTypes.bool,
+    showOnFocus: PropTypes.bool,
+    allowTab: PropTypes.bool,
+    transition: PropTypes.string,
+    duration: PropTypes.number,
+    keys: PropTypes.object,
+    delay: PropTypes.object,
+
+    // Callbacks
+    onChange: PropTypes.func,
+    onShow: PropTypes.func,
+    onHide: PropTypes.func,
+    onNoResults: PropTypes.func,
+    onLabelSelect: PropTypes.func,
+    onLabelRemove: PropTypes.func,
+    onLabelCreate: PropTypes.func,
+    onAdd: PropTypes.func,
+    onRemove: PropTypes.func,
   };
 
   componentDidMount() {
     this.element = $(this.refs.select);
-    this.element.dropdown();
+    this.element.dropdown({
+      // Settings
+      action: this.props.action,
+      allowAdditions: this.props.allowAdditions,
+      allowCategorySelection: this.props.allowCategorySelection,
+      apiSettings: this.props.apiSettings,
+      fields: this.props.fields,
+      forceSelection: this.props.forceSelection,
+      glyphWidth: this.props.glyphWidth,
+      label: this.props.label,
+      match: this.props.match,
+      maxSelections: this.props.maxSelections,
+      on: this.props.on,
+      placeholder: this.props.placeholder,
+      saveRemoveData: this.props.saveRemoteData,
```

Comment:
Looking at [Dropdown Settings - Remote Settings](http://semantic-ui.com/modules/dropdown.html#remote-settings) this defaults to true, meaning form values persist to session storage. Though I haven't actually seen it function yet, it'd be nice to have in the future.